mne/io/fiff/raw.py

```diff
@@ -144,7 +144,6 @@ def _read_raw_file(self, fname, allow_maxshield, preload,
                 endings += tuple([f'{e}.gz' for e in endings])
                 check_fname(fname, 'raw', endings)
             # filename
-            fname = op.realpath(fname)
```

Comment:
I tracked it down to a comment years ago:

https://github.com/mne-tools/mne-python/pull/1223#discussion-diff-11646059

I think we should at least keep `abspath` in there in case someone is using a relative path and then `os.chdir`s.

I can add a quick regression test for this use case, make sure it fails, and push a commit.

Reply:
sounds good, although I wonder how could someone `os.chdir` in the middle of an operation where a multi-part raw file was being loaded?

Reply:
For `preload=False` the filename gets stored and accessed/used on the fly later